api/src/main/java/io/lumeer/engine/api/dto/Roles.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *  
+ * Copyright (C) since 2016 the original author or authors.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+package io.lumeer.engine.api.dto;
+
+import io.lumeer.engine.api.LumeerConst;
+import io.lumeer.engine.api.data.DataDocument;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.DataFormatException;
+/**
+ * @author <a href="mailto:martin.glova@lumeer.io">Martin Glova</a>
+ */
+public class Roles extends DataDocument {
+   public Roles(final DataDocument data, final String resource) throws DataFormatException {
```

Comment:
DTO objects should look like others. So it have to contains string,integer and other needed fields..